native-oci-generative-ai/src/main/java/com/oracle/labs/GenerateTextExample.java

```diff
@@ -0,0 +1,140 @@
+import com.oracle.bmc.retrier.RetryConfiguration;
+
+/**
+ * This class provides an example of how to use OCI Generative AI Service to
+ * generate text.
+ * <p>
+ * The Generative AI Service queried by this example will be assigned:
+ * <ul>
+ * <li>an endpoint url defined by constant ENDPOINT</li>
+ * <li>
+ * The configuration file used by service clients will be sourced from the
+ * default
+ * location (~/.oci/config) and the CONFIG_PROFILE profile will be used.
+ * </li>
+ * </ul>
+ * </p>
+ */
+//@Slf4j
+public class GenerateTextExample {
+  private static final String ENDPOINT = "https://inference.generativeai.us-chicago-1.oci.oraclecloud.com";
+  private static final Region REGION = Region.US_CHICAGO_1;
+  private static final String CONFIG_LOCATION = "~/.oci/config";
+  // TODO: Please update config profile name and use the compartmentId that has
+  // policies grant permissions for using Generative AI Service
+  private static final String CONFIG_PROFILE = "DEFAULT";
+  private static final String COMPARTMENT_ID = "ocid1.compartment.oc1..aaaaaaaaul6555vm7yt6qm7tmsq64yvivkidstxajilzcjv2laaki46rtdzq";
+
+  /**
+   * The entry point for the example.
+   *
+   * @param args Arguments to provide to the example. This example expects no
+   *             arguments.
+   * @throws java.lang.Exception
+   */
+  public static void main(String[] args) throws Exception {
+    if (args.length > 0) {
+      throw new IllegalArgumentException(
+          "This example expects no argument");
+    }
+    // Configuring the AuthenticationDetailsProvider. It's assuming there is a
+    // default OCI config file
+    // "~/.oci/config", and a profile in that config with the name defined in
+    // CONFIG_PROFILE variable.
+    final ConfigFileReader.ConfigFile configFile = ConfigFileReader.parse(CONFIG_LOCATION, CONFIG_PROFILE);
+    final AuthenticationDetailsProvider provider = new ConfigFileAuthenticationDetailsProvider(configFile);
+
+    // Set up Generative AI client with credentials and endpoint
+    ClientConfiguration clientConfiguration = ClientConfiguration.builder()
+        .readTimeoutMillis(240000)
+        .retryConfiguration(RetryConfiguration.NO_RETRY_CONFIGURATION)
+        .build();
+
+    final GenerativeAiInferenceClient generativeAiInferenceClient = GenerativeAiInferenceClient.builder()
+          .configuration(clientConfiguration)
+          .build(provider);
+    generativeAiInferenceClient.setEndpoint(ENDPOINT);
+    generativeAiInferenceClient.setRegion(REGION);
+
+    // Build generate text request, send, and get response
+    CohereLlmInferenceRequest llmInferenceRequest = CohereLlmInferenceRequest.builder()
+        .prompt(
+            "Generate a job description for a data visualization expert with the following three qualifications only:\n1) At least 5 years of data visualization expert\n2) A great eye for details\n3) Ability to create original visualizations")
```

Comment:
Use a text block (and, unlike me, an IDE for formatting :wink:)
```suggestion
        .prompt("""
            Generate a job description for a data visualization expert with the following three qualifications only:
            1) At least 5 years of data visualization expert
            2) A great eye for details
            3) Ability to create original visualizations")
        """)
```